llvm/lib/Target/DirectX/DirectXTargetMachine.cpp

```diff
@@ -50,6 +51,7 @@ extern "C" LLVM_EXTERNAL_VISIBILITY void LLVMInitializeDirectXTarget() {
   initializeDXContainerGlobalsPass(*PR);
   initializeDXILOpLoweringLegacyPass(*PR);
   initializeDXILTranslateMetadataLegacyPass(*PR);
+  initializeDXILMetadataAnalysisWrapperPassPass(*PR);
```

Comment:
We shouldn't need this - this pass isn't part of the DirectX target.

Reply:
> We shouldn't need this - this pass isn't part of the DirectX target.

Deleted.